ValidarTipoArchivo.js

```diff
@@ -0,0 +1,16 @@
+/*Validar Tipo de Archivo*/
+    function ValidarImagen(input){
+    let imagen = input.files[0]
+    let arr = ['image/jpg','image/jpeg','image/png']
+
+    if (arr.indexOf(imagen.type) != -1 ){
+    console.log('es valido')
+}
+    else{
+    input.value = ''
+    console.log('no es valido')
+    alert('No es Valido, Selecciona otro archivo')
+}
```

Comment:
en este caso realmente lo que importa saber es cuando no es valido para mostar el mensaje de alerta, si es valido no se hace nada, entonces podemos cambiar el codigo para solo comprobar cuando no es valido, pudes usar la [negacion](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_NOT)

```suggestion
   if (!arr.indexOf(imagen.type) != -1 ) {
     input.value = ''
     alert('No es Valido, Selecciona otro archivo')
   }
```